evalml/model_understanding/prediction_explanations/_user_interface.py

```diff
@@ -202,13 +202,13 @@ def make_dict(self, shap_values, normalized_values, pipeline_features, top_k, in
         return {"explanations": json_output}
 
 
-def _make_single_prediction_shap_table(pipeline, input_features, y, index_to_explain, top_k=3,
+def _make_single_prediction_shap_table(pipeline, pipeline_features, index_to_explain, top_k=3,
                                        include_shap_values=False, output_format="text"):
     """Creates table summarizing the top_k_features positive and top_k_features negative contributing features to the prediction of a single datapoint.
 
     Arguments:
         pipeline (PipelineBase): Fitted pipeline whose predictions we want to explain with SHAP.
-        input_features (pd.DataFrame): Dataframe of features - needs to correspond to data the pipeline was fit on.
+        pipeline_features (pd.DataFrame): Dataframe of features computed the pipeline.
```

Comment:
wording here is confusing

Reply:
lol, perhaps : "DataFrame of the features used to fit the pipeline"?

Reply:
Changed it "Dataframe of features computed by the pipeline". It's the result of `compute_estimator_features` not the data passed in by the user.